app/config.py

```diff
@@ -176,6 +176,9 @@ class Config(object):
     CELERY_TASK_SERIALIZER = 'json'
     # on reporting worker, restart workers after each task is executed to help prevent memory leaks
     CELERYD_MAX_TASKS_PER_CHILD = os.getenv('CELERYD_MAX_TASKS_PER_CHILD')
+    # we can set celeryd_prefetch_multiplier to be 1 for celery apps which handle only long running tasks
+    if os.getenv('CELERYD_PREFETCH_MULTIPLIER'):
```

Comment:
Do we need this if statement? The CELERYD_MAX_TASKS_PER_CHILD option is not using one.

Reply:
I'm not sure what is different about the max tasks per child setting but if we don't include this statement then we get the following error when I run `./scripts/run_celery.sh` locally

```
File "/Users/davidmcdonald/.virtualenvs/notifications-api/lib/python3.6/site-packages/celery/worker/__init__.py", line 381, in setup_defaults
    'prefetch_multiplier', prefetch_multiplier,
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
you could have it default to `4` rather than `None` i guess using the optional arg in os.getenv

Reply:
I considered this and then decided I didn't want to replicate a setting in celery just in case celery later on decides to change it and we aren't following and update ours too